### DISCUSSION TITLE
Idea for tags plugin - alternate tag index layout

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [squidfunk/mkdocs-material#5135] [Ideas]
The [built-in tags plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#built-in-tags-plugin) allows to define a file to render a [tags index](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#tags-file).

This prints the tag label followed by the pages which contains that tag index.

![image](https://user-images.githubusercontent.com/95432492/222716648-a8e236b8-f4f0-4479-b694-a201b3b59d81.png)

It would be cool if there was the option to change how the tag index is rendered. For example a table with a row per page which displays all the labels for that page:

[filename][ title] [label1][label2]...

**Comment by squidfunk** [comment]
We're planning to make the tags rendering configurable by providing a custom template via regular theme extension in one of the next releases. You can open a change request for this if you like.

**Reply by SoumayaMauthoorMOJ** [reply]
Done see https://github.com/squidfunk/mkdocs-material/issues/5137 :-)
